pkg/dataloader/featuregateloader/featuregateloader.go

```diff
@@ -87,7 +87,7 @@ func (l *FeatureGateLoader) getTargetReleases() ([]string, error) {
 
 		v, err := version.NewVersion(release.Release)
 		if err != nil {
-			return nil, errors.Wrapf(err, "error parsing release version %s", release)
+			return nil, errors.Wrapf(err, "error parsing release version %s", release.Release)
```

Comment:
this hasn't been touched otherwise in this PR but for some reason it was failing in my local tests so i fixed it.